src/components/VerticalNavigation/VerticalNavigationBrand.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const VerticalNavigationBrand = props => {
+  const { title, href, iconImg, children } = props;
+  // The img prop is just a shorthand for the titleImg prop.
+  // When also using iconImg, it is less confusing to pass titleImg instead of img.
+  const titleImg = props.titleImg || props.img;
+
+  const brandChildren = children || (
+    <span>
+      {iconImg && (
+        <img className="navbar-brand-icon" src={iconImg} alt={title} />
+      )}
+      {titleImg && (
+        <img className="navbar-brand-name" src={titleImg} alt={title} />
+      )}
+      {!titleImg && title && <span className="navbar-brand-txt">{title}</span>}
```

Comment:
Oh, this is where that class is being used. Is this variation viewable in the storybook?

Reply:
@jgiardino, yes, this is the "Patternfly React" text that appears at the top in all the Vertical Nav stories except the last one (custom masthead). If you render a `<VerticalNavigation.Masthead title="..." />` it will use that span.navbar-brand-txt

Reply:
Ah, I just realized that the link I had posted hid the storybook navigation so you couldn't switch to the other stories.  Sorry about that!  You can see them all at https://mturley.github.io/patternfly-react/